### ISSUE TITLE
Few Questions on the code

### ISSUE THREAD

**Issue by dssrgu** [blackbox-bandits/blackbox-attacks-bandits-priors#1]
Hi,
While trying to reproduce your results in the paper "Prior Convictions: Black-box Adversarial Attacks with Bandits and Priors", I came to have 2 questions.

1. The paper says that 10000 random images were chosen from ImageNet. Is there anyway to attack the same image set the paper used to attack?(like random seed?)

2. In Pytorch document(https://pytorch.org/docs/stable/torchvision/models.html), it says that all pre-trained models use normalized input images(by torchvision.transforms.Normalize). But your code does not use normalizing for the images. Testing on validation set, I found that model accuracy on this code is 68.6%. After adding normalizing, model accuracy rose to 77.2%, which is expected for Inception v3. Is your code different from the paper? Did you use normalizing while producing results for the paper?

**Comment by ZiangYan** [comment]
Hi @dssrgu , have you solved the normalization problem? I've found the same problem...

**Comment by ZiangYan** [comment]
Sorry, my mistake.

These problems are all fixed in the new official repo: https://github.com/MadryLab/blackbox-bandits
